js/controller/foldercontroller.js

```diff
@@ -113,8 +113,8 @@ define(function(require) {
 	function showFolder(account, folder, loadFirstMessage) {
 		loadFirstMessage = loadFirstMessage !== false;
 		Radio.ui.trigger('search:set', '');
-		Radio.ui.trigger('content:loading', t('mail', 'Loading {folder}', {
-			folder: folder.get('name')
+		Radio.ui.trigger('content:loading', t('mail', 'Loading folder {folder}', {
+			folder: '»' + folder.get('name') + '«'
```

Comment:
FYI I've added those here on purpose because otherwise translators have to insert them in the translated string for every language. This way we can also easily remove the quotes if we device we don't like them and can still use the old translation strings.